linux_os/guide/services/radius/group.yml

```diff
@@ -0,0 +1,11 @@
+documentation_complete: true
+
+title: 'Radius'
+
+description: |-
+    Remote Authentication Dial-In User Service (RADIUS) is a networking
+    protocol, operating on port 1812 that provides centralized
+    Authentication, Authorization, and Accounting (AAA or Triple A)
+    management for users who connect and use a network service. 
+
+platform: machine
```

Comment:
Can a Radius server run in a container?

Reply:
Good call, it can run in a container!

How would I note that here?  Just drop `platform`?

Reply:
@jcpunk yeah... just drop platform.

Reply:
can do.